scripts/cmake-configurator/parser/test_cmakeparser.py

```diff
@@ -0,0 +1,45 @@
+from cmake_parser import cmake_parser
+from unittest import TestCase
+
+file_lines_generic = """
+# KEY:TYPE=VALUE
+# KEY is the name of a variable in the cache.
+# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
+# VALUE is the current value for the KEY.
+
+########################
+# EXTERNAL cache entries
+########################
+
+//Build unit tests for the target arch
```

Comment:
- Missing multi-line comments on the top of variables
- Missing variables without comments on the top

Reply:
Resolved